includes/Form.class.php

```diff
@@ -165,6 +165,18 @@ function process($params) {
 		}
 		$email = $_POST["email"];
 		$phone = isset($_POST["phone"]) ? $_POST["phone"] : "";
+		$lists = (isset($_POST["nlbox"]) && $_POST["nlbox"]) ? $_POST["nlbox"] : array();
+		if (isset($_SESSION["image_random_value"])) {

```

Comment:
Okay have a look at the changes now. They are related to [this change in the WordPress plugin repo](https://github.com/ActiveCampaign/app-wordpress/pull/6). Now we detect if a captcha field is part of the form, and if so we force that check. So there's no way to bypass the captcha check just by making the API request outside of the browser.